Dockerfile.test.gpu

```diff
@@ -159,16 +167,21 @@ RUN if [[ ${MXNET_PACKAGE} != "mxnet-nightly-cu"* ]]; then \
     fi
 
 # Prefetch Spark MNIST dataset.
-RUN mkdir -p /work
-RUN mkdir -p /data && wget --progress=dot:mega https://horovod-datasets.s3.amazonaws.com/mnist.bz2 -O /data/mnist.bz2
+RUN mkdir -p /work && mkdir -p /data && wget --progress=dot:mega https://horovod-datasets.s3.amazonaws.com/mnist.bz2 -O /data/mnist.bz2
 
 # Prefetch Spark Rossmann dataset.
-RUN mkdir -p /work
 RUN mkdir -p /data && wget --progress=dot:mega https://horovod-datasets.s3.amazonaws.com/rossmann.tgz -O - | tar -xzC /data
 
 # Prefetch PyTorch datasets.
 RUN wget --progress=dot:mega https://horovod-datasets.s3.amazonaws.com/pytorch_datasets.tgz -O - | tar -xzC /data
 
+# Patches (leveraging cached layers above)
```

Comment:
What do you mean with "leveraging cached layers"? In which sense is this comment specifically important here?

Reply:
Not particularly important, so will remove.